mne/io/boxy/tests/test_boxy.py

```diff
@@ -0,0 +1,296 @@
+# Authors: Kyle Mathewson, Jonathan Kuziek <kuziek@ualberta.ca>
+#
+# License: BSD (3-clause)
+
+import os
+
+import numpy as np
+from numpy.testing import assert_allclose, assert_array_equal
+import scipy.io as spio
+
+import mne
+from mne.datasets.testing import data_path, requires_testing_data
+
+
+@requires_testing_data
+def test_boxy_load():
+    """Test reading BOXY files."""
+    # Determine to which decimal place we will compare.
+    thresh = 1e-10
+
+    # Load AC, DC, and Phase data.
+    boxy_file = os.path.join(data_path(download=False),
+                             'BOXY', 'boxy_0_40_recording',
+                             'boxy_0_40_notriggers_unparsed.txt')
+
+    boxy_data = mne.io.read_raw_boxy(boxy_file, verbose=True).load_data()
+
+    # Test sampling rate.
+    assert boxy_data.info['sfreq'] == 62.5
+
```

Comment:
Can you test the returned types here?

Reply:
Something analogous to
https://github.com/mne-tools/mne-python/blob/40872a8b836a86a48d013a1654b2bfffb59ea5b8/mne/io/nirx/tests/test_nirx.py#L308

Reply:
```
    #Test the returned types
    assert 'fnirs_fd_phase' in raw 
    assert 'fnirs_fd_dc_amplitude' in raw 
    assert 'fnirs_fd_ac_amplitude' in raw 
```